src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -79,7 +79,7 @@
  * @see RoutingAdviceTrait
  * @see ValidationAdviceTrait
  */
-public interface AdviceTrait {
+public interface AdviceTrait extends StatusConverter {
```

Comment:
Ah, I didn't realize that the interface is package private. I'd say we can leave it at is.